package.json

```diff
@@ -11,12 +11,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@sendgrid/mail": "^7.6.1",
```

Comment:
Do we need to install this module to get the mailer to work? If not necessary, let's delete it. (I don't see the use case in the code base at least, that's why I comment but I could be wrong, just double checking)